verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -1,12 +1,21 @@
 import React from "react";
 import axios from 'axios';
 import LinelistCaseRow, { Case } from "./LinelistCaseRow";
-import Table from 'react-bootstrap/Table';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TablePagination from '@material-ui/core/TablePagination';
+import TableRow from '@material-ui/core/TableRow';
```

Comment:
nit: do we care about sorting imports?

Reply:
I think we do, but I'm not sure. I'll fix this and reinvigorate my search for an auto import sorter.

Reply:
https://marketplace.visualstudio.com/items?itemName=amatiasq.sort-imports ?

Reply:
Ideally it'd be on the eslint side, but this is a good find.